webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/detail/AnnotationFeatureForm.java

```diff
@@ -369,7 +260,7 @@ protected void onUpdate(AjaxRequestTarget aTarget)
                     return;
                 }
                 selectedTag = (forwardAnnotationText.getModelObject() == null ? ""
-                    : forwardAnnotationText.getModelObject().charAt(0)) + selectedTag;
+                        : forwardAnnotationText.getModelObject().charAt(0)) + selectedTag;
```

Comment:
Look below at line 376/270.
> aTarget.add(featureEditorPanelContent.get(0));

_org.apache.wicket.Component get(int index)_ is deprecated and marked for deletion for WICKET8.